src/doom/am_map.c

```diff
@@ -1278,6 +1278,11 @@ static void AM_drawFline_Smooth(fline_t* fl, int color)
         X0 = X1;
         X1 = Temp;
     }
+#ifdef CRISPY_TRUECOLOR
+    /* We have already applied the colormap lookup in AM_LevelInit so can directly copy the values */
+#undef PUTDOT
```

Comment:
The `#undef` is a bit too ugly. Could we get around without it, i.e. with a single definition of `PUTDOT`?